python/GafferArnold/ArnoldTextureBake.py

```diff
@@ -257,6 +272,8 @@ def __init__( self, name = "ArnoldTextureBake" ) :
 		self["tasks"] = Gaffer.IntPlug( "tasks", defaultValue = 1 )
 		self["cleanupIntermediateFiles"] = Gaffer.BoolPlug( "cleanupIntermediateFiles", defaultValue = True )
 
+		self["selectUdims"] = Gaffer.StringPlug( "selectUdims", defaultValue = "" )
```

Comment:
Let's just call this `udims` - where there's an obvious noun it usually makes a better name than a verb does. I think the most natural place in the list of plugs would be just after `uvSet` as well, so that we're keeping all the "what to bake" options together.